examples/egress/egress-policy.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: "projectcalico.org/v1"
```

Comment:
As mentioned elsewhere, let's stick this at `projectcalico.org/v1alpha1` for now while we work out the correct API and semantics.  We can promote to v1 at a later date.